examples/with-cookie-auth-fauna/utils/fauna-auth.js

```diff
@@ -8,11 +8,21 @@ export const serverClient = new faunadb.Client({
 })
 
 // Used for any authed requests.
-export const faunaClient = (secret) =>
+export const faunaClient = (secret, domain) =>
   new faunadb.Client({
-    secret,
+    secret: secret,
+    domain: domain
```

Comment:
That sounds like a good idea

Reply:
I considered that approach as well, but decided that clients may ultimately want to pass in a domain themselves. Instead I opted to export the convenience method which handles the fallback logic, but still allow for overriding should a customer choose.

That said, I can be convinced otherwise if customers tend to just stick with defaults, or whatever their environment vars deploy with.

Reply:
I think we can assume that you would use the same domain for all fauna clients in one example.